doc/api/n-api.md

```diff
@@ -3134,6 +3134,35 @@ overhead in creating/storing strings with this method.
 The JavaScript `string` type is described in
 [Section 6.1.4][] of the ECMAScript Language Specification.
 
+#### `napi_set_named_property_len`
```

Comment:
New APIs should have the prefix `node_api_*`, i.e. this should be renamed to `node_api_set_named_property_len`.